README.md

```diff
@@ -41,102 +41,76 @@ Python versions currently provided are:
 * [python-3.8](3.8)
 
 RHEL versions currently supported are:
-* RHEL7
+* RHEL 7 ([catalog.redhat.com](https://catalog.redhat.com/software/containers))
+* RHEL 8 ([catalog.redhat.com](https://catalog.redhat.com/software/containers))
```

Comment:
This link leads to a very general page. How about using `https://catalog.redhat.com/software/containers/search`?

Reply:
You are right, thank you.